Foundation/NSAttributedString.swift

```diff
@@ -42,73 +58,87 @@ open class NSAttributedString: NSObject, NSCopying, NSMutableCopying, NSSecureCo
     open func mutableCopy(with zone: NSZone? = nil) -> Any {
         NSUnimplemented()
     }
-    
+
+    /// The character contents of the receiver as an NSString object.
     open var string: String {
         return _string._swiftObject
     }
-    
-    open func attributes(at location: Int, effectiveRange range: NSRangePointer) -> [String : Any] {
+
+    /// Returns the attributes for the character at a given index.
+    open func attributes(at location: Int, effectiveRange range: NSRangePointer) -> [NSAttributedStringKey : Any] {
```

Comment:
The `NSRangePointer` should be optional: https://developer.apple.com/documentation/foundation/nsattributedstring/1415682-attributes

Reply:
fixed